### ISSUE TITLE
Deep Adaptive Input Normalization Layers in Keras 

### ISSUE THREAD

**Issue by losDaniel** [keras-team/keras#13996]
Hello, I am wondering if there are any plans to incorporate Deep Adaptive Input Normalization (DAIN) layers in Keras this year. 

The layers were first put forward in this paper [Passalis, Nikolaos, et al. "Deep Adaptive Input Normalization for Price Forecasting using Limit Order Book Data." arXiv preprint arXiv:1902.07892 (2019).](https://arxiv.org/pdf/1902.07892.pdf)

It seems like it would be a welcome addition to the normalization layers offered by Keras which currently only include `BatchNormalization` which can be sub-optimal for modeling time-series data with varying sequence lengths.

**Comment by lorrp1** [comment]
Any update? I’m interest as well.

**Comment by Mathbone** [comment]
I would like to work on this as a final project for a neural networks course. Having a final PR accepted is not strictly necessary for the project, but I would like to move it in that direction for my own education. 

I'm reading through the contributor guidelines, but is there a process for putting an incomplete PR where others can also work on it?
